src/components/layout/LayoutProvider/__stories__/Layout.mdx

```diff
@@ -157,6 +155,26 @@ export const App = () => {
 };
 ```
 
+## Box
+
+The `Box` component is a developer friend and basic block to build other components. Aware about spacing, its own sizes and most commonly used CSS properties.
```

Comment:
Is this a11y friendly?

Reply:
it's support all html attributes if you question about this:
https://github.com/gravity-ui/uikit/pull/1121/files#diff-048664dcf8c222b5be90577dba7bc1dfe668567e583acbe28dfdf9ec20cd0406R12-R21